codebuild/codebuild.config

```diff
@@ -74,17 +74,9 @@ snippet: UbuntuBoilerplate2XL
 env: S2N_LIBCRYPTO=openssl-1.0.2-fips BUILD_S2N=true TESTS=valgrind GCC_VERSION=6
 
 # SAW tests
-[CodeBuild:s2nSawHmac]
-snippet: UbuntuBoilerplate2XL
-env: TESTS=sawHMAC SAW=true GCC_VERSION=NONE
-
-[CodeBuild:s2nSawDrbg]
-snippet: UbuntuBoilerplate2XL
-env: TESTS=sawDRBG SAW=true GCC_VERSION=NONE
-
-[CodeBuild:s2nSawSike]
-snippet: UbuntuBoilerplate2XL
-env: TESTS=sawSIKE SAW=true GCC_VERSION=NONE
+[CodeBuild:s2nSawHmacPlus]
```

Comment:
Q: What  does `s2nSawHmacPlus` mean here?

Reply:
This defines a new CodeBuild job, created on CFN script run.  I'm trying to make it obvious which jobs have combined runs.